### PR TITLE
De 2151 bronto contact update fix

### PR DESCRIPTION
**Pull request by TurnerMooreDavis**
Adds a new method `change_status` which can be called on a contact and ACTUALLY updates the status.  This was needed because the `add_or_update` endpoint will NOT update the status as described here: http://dev.bronto.com/api/soap/functions/update/addorupdatecontacts-2/

This method will be called in Spoonclassic once this PR is merged into master:
https://github.com/Spoonflower/spoonflower/pull/1770

### CODE REVIEW

**Review by katiebuilds — APPROVED** [review]

**Comment by ghost** [comment]
Looks good!

File: lib/bronto/contact.rb [hunk 1]
```diff
@@ -36,6 +36,17 @@ def self.save(*objs)
...
+    
+    def change_status(status)
+      api_key = self.api_key
+      
+      resp = request(:update, {self.class.plural_class_name => [{:id => self.id, :status => status}]})
+
+      self.errors.clear
+      result = resp[:return][:results]
+      self.errors.add(result[:error_code], result[:error_string]) if result[:is_error]
+      return self
```
Comment (tomgordon) [inline lib/bronto/contact.rb]: The return keyword isn't needed here.

**Review by tomgordon — APPROVED** [review]
OK but look at comment.

File: lib/bronto/contact.rb (hunk 1 above)
Comment (TurnerMooreDavis) [reply lib/bronto/contact.rb]: 👍

**Pull request merged**